winml/test/common/fileHelpers.cpp

```diff
@@ -0,0 +1,66 @@
+﻿#include "testPch.h"
+#include "fileHelpers.h"
+#include "winrt/Windows.Media.h"
+#include "winrt/Windows.Storage.h"
+
+EXTERN_C IMAGE_DOS_HEADER __ImageBase;
+
+using namespace winrt;
+using namespace winrt::Windows::AI::MachineLearning;
+using namespace winrt::Windows::Graphics::Imaging;
+using namespace winrt::Windows::Storage;
+
+namespace FileHelpers
+{
+    std::wstring GetModulePath()
+    {
+        std::wstring val;
+        wchar_t modulePath[MAX_PATH] = { 0 };
+        GetModuleFileNameW((HINSTANCE)&__ImageBase, modulePath, _countof(modulePath));
+        wchar_t drive[_MAX_DRIVE];
+        wchar_t dir[_MAX_DIR];
+        wchar_t filename[_MAX_FNAME];
+        wchar_t ext[_MAX_EXT];
+        errno_t err = _wsplitpath_s(modulePath, drive, _MAX_DRIVE, dir, _MAX_DIR, filename, _MAX_FNAME, ext, _MAX_EXT);
+
+        val = drive;
+        val += dir;
+
+        return val;
+    }
+
+    std::wstring GetWinMLPath()
+    {
+        bool inboxDll = false;
+        // if (SUCCEEDED(WEX::TestExecution::RuntimeParameters::TryGetValue(L"inbox", inboxDll)) && inboxDll)
+        // {
+        //     return L"";
+        // }
```

Comment:
There's no RuntimeParameters in googletest, I'll add command line arguments parsing in a later PR